dodos/noisepage.py

```diff
@@ -67,10 +69,10 @@ def task_noisepage_build():
         "actions": [
             lambda: os.chdir(BUILD_PATH),
             # Configure NoisePage.
-            "./cmudb/build/configure.sh release",
+            "doit np_config --build_type=release",
             # Compile NoisePage.
-            "make -j world-bin",
-            "make install-world-bin",
+            "doit np_build",
+            "doit hutch_install",
```

Comment:
Hm.. Do we _always_ want to build `hutch`? Or does it make sense to split this out, especially considering that extensions can break each other?

I would argue that this should be split out in the same way that `hypopg` is. This ties in to the above comment about `shared_preload_libraries`.